CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+- [`89b6d96`](http://github.com/epfl-si/startpage/commit/89b6d96ab19d7cefe7a90d5be1b65fcef977f2e4") `2023-11-15` [feature] CI with GitHub Actions (#5)
+- [`e5f7080`](http://github.com/epfl-si/startpage/commit/e5f7080dc4865aff2a4893a9fd2687e603df8030") `2023-11-10` [feature] add favicon(s) (#23)
+- [`46ea0d8`](http://github.com/epfl-si/startpage/commit/46ea0d83262486bad27a16cde5688f82027c669b") `2023-11-10` [feature] magnifying glass icon (#22)
+- [`38fc47b`](http://github.com/epfl-si/startpage/commit/38fc47b3902918949cded9b49dbd2575f5430df3") `2023-11-10` [feature] ensure minimal node version (>20) (#21)
+- [`8e8bacb`](http://github.com/epfl-si/startpage/commit/8e8bacbb0f6c935c3c11dcf96c9d1cdd4fac80f0") `2023-11-08` [feature] smart search
+- [`8d1541b`](http://github.com/epfl-si/startpage/commit/8d1541bda8b6ee9eb30475b7528e7534ff5a3bd3") `2023-11-06` [fix] Code formatting (#6)
+- [`56e6022`](http://github.com/epfl-si/startpage/commit/56e6022692ed58ee1ad582367cca2cb3fad63676") `2023-11-06` [feature] eslint (#4)
+- [`e6a70f3`](http://github.com/epfl-si/startpage/commit/e6a70f3c2aea918c1c63e0cb15f0d5a1cf823e32") `2023-11-06` [feature] prettier (#3)
+- [`fab8cb5`](http://github.com/epfl-si/startpage/commit/fab8cb5baea6a91775a4bdf2a87de6a928ba56c8") `2023-11-03` [feature] pnpm (#2)
+- [`a44c7d8`](http://github.com/epfl-si/startpage/commit/a44c7d8c0afffdb290ee81eb804cf410a19105dd") `2023-11-01` [feature] catalog page
+- [`c106e32`](http://github.com/epfl-si/startpage/commit/c106e3285f0ae757886924af42fa8f215a1651a0") `2023-11-01` [fix] nodemon for ejs file
+- [`421d94f`](http://github.com/epfl-si/startpage/commit/421d94f201c8f9c242a135bda9de110e04b01bf3") `2023-11-01` [feature] data from EPFL Arrival
+- [`511a373`](http://github.com/epfl-si/startpage/commit/511a37330a18dd68a6a4fc6ec8f1cc8d17833a54") `2023-11-01` [feature] CHANGELOG, CONTRIBUTING and DEVELOPMENT
+- [`d1b54c6`](http://github.com/epfl-si/startpage/commit/d1b54c6ae03d660621a23f366e99a441251c6a6b") `2023-11-01` [feature] Add nodemon
+- [`c36296e`](http://github.com/epfl-si/startpage/commit/c36296e40b88af544ab72c3ab5ef20075e65ee65") `2023-11-01` [feature] Separate head and footer in partials
+- [`909e92d`](http://github.com/epfl-si/startpage/commit/909e92d91d1e7aaac4896ed31df4f20065ebe5ff") `2023-11-01` [feature] Logging what's typed in the search bar
+- [`ff3fee2`](http://github.com/epfl-si/startpage/commit/ff3fee25cd2d5f20f7fc7c80a43e9fef16aaaf15") `2023-11-01` [feature] Basic search bar
+- [`f83faeb`](http://github.com/epfl-si/startpage/commit/f83faeb830934f93d3216359856bd49c54d4d0fa") `2023-11-01` [feature] Hello world EJS & Express
+- [`ee9aadd`](http://github.com/epfl-si/startpage/commit/ee9aadd179257df53dbeb99621db10a89bafda00") `2023-11-01` [feature] Add editor config
+- [`570b00d`](http://github.com/epfl-si/startpage/commit/570b00ded0180c360a8e824340e5e6be60774aff") `2023-11-01` [hello] startpage.epfl.ch%
```

Comment:
Percentage symbol at the end. zsh?

Reply:
Its way to say there's no new line after the command. Fixed.